tests/puppeteer/campaigns/smoke/crawlingFO.js

```diff
@@ -0,0 +1,32 @@
+// Importing pages
+const PAGE = require('../../pages/FO/home');
```

Comment:
```suggestion
const HomePage = require('../../pages/FO/home');
```
This page is the HomePage, and this variable holds the class, so same format.